package.json

```diff
@@ -94,6 +94,7 @@
     "heimdalljs-logger": "^0.1.10",
     "http-proxy": "^1.18.1",
     "inflection": "^1.13.1",
+    "inquirer": "^8.2.1",
```

Comment:
Currently we get inquirer via console-ui, can we refactor to leverage that?

Reply:
We could, but `console-ui` still depends on v6, and v6 doesn't support providing fixed answers, which I'm using during tests. This would also need an API change in `console-ui` I think.
What's the benefit of using `ui.prompt` instead of using `inquirer` directly?
Seems like an unnecessary abstraction at first glance, but maybe I'm missing some context.

Reply:
I second the "unnecessary abstraction" assessment. `ui.prompt`  / `console-ui` does not have very many docs (and it's not clear what's it is using under its APIs), and folks wanting to contribute to the interactive flow in ember-cli will appreciate knowing that `inquirer` is used, which has _toooons of docs_.

Reply:
Well, we do need to update/clean-up the console-ui stuff more generally, but I think this is fine for now (other than that we have two or more copies of inquirer in our stack, which is a bit annoying).